.github/workflows/release.yml

```diff
@@ -126,3 +126,9 @@ jobs:
           --project="${{ env.DEV_PROJECT_ID }}" \
           --region="${{ env.DEV_REGION }}" \
           --image="${{ env.CONTAINER_REGISTRY }}/jvsctl:${DOCKER_TAG}-amd64"
+
+        # Update Prober job.
+        gcloud run jobs update jvs-prober \
```

Comment:
I don't know if this is correct. This is a _matrix_ job, so it runs once per each matrix entry defined on L112. This will update the prober 4x, which is not what we want.

Reply:
The items inside the matrix are all services deployed via [1], and need to be updated via "gcloud run services update", while prober is a job deployed via [2] and need to be updated by "gcloud run jobs update". Let me know whether that is how things are deployed in dev project tycho-jvs-dev-e06f, happy to update accordingly :D

[1] https://github.com/abcxyz/jvs/blob/main/terraform/e2e/main.tf#L57
[2] https://github.com/abcxyz/jvs/blob/main/terraform/modules/monitoring/prober.tf#L17

Reply:
Thanks for the details, good to learn about the matrix strategy :D Agree that it is better to split it to a separate job in this case.

Updated. PTAL :D